source/renderer/app/containers/settings/categories/SecuritySettingsPage.tsx

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import { observer } from 'mobx-react';
 import { useDiscreetModeFeature } from '../../../features/discreet-mode';
 import SecuritySettings from '../../../components/settings/categories/SecuritySettings';
+import { useSendPageNavigationEvent } from '../../../analytics/useSendPageNavigationEvent';
 
 const SecuritySettingsPage = () => {
+  useSendPageNavigationEvent('Security Settings');
```

Comment:
Here I can just use `useAnalytics` hook and call the metod from object it exposes